### DISCUSSION TITLE
ct2 primeline/distil-whisper-large-v3-german -> input shape issue

### DISCUSSION THREAD

**Discussion by kvrban** [SYSTRAN/faster-whisper#831] [General]
I try to use the German Speech Recognition model https://huggingface.co/primeline/distil-whisper-large-v3-german

`ct2-transformers-converter --model primeline/distil-whisper-large-v3-german --output_dir primeline/distil-whisper-large-v3-german
`

But when you load the model:
`ValueError: Invalid input features shape: expected an input with shape (1, 128, 3000), but got an input with shape (1, 80, 3000) instead`

**Comment by trungkienbkhn** [comment]
@kvrban , hello. You can refer this [issue](https://github.com/SYSTRAN/faster-whisper/issues/593), should copy 'preprocess_config.json' file when converting model.

**Reply by kvrban** [reply]
thanks, that worked. unfortunately this model is worse in the german translation than the original
